packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -124,17 +125,30 @@ class _FlagList extends StatelessWidget {
               mainAxisAlignment: MainAxisAlignment.spaceBetween,
               children: <Widget>[
                 Expanded(
```

Comment:
this could be a good first screen to add a golden image test for as I think it is now done and won't be iterated on too much going forward.